portal-wire-protocol.md

```diff
@@ -123,6 +123,7 @@ nodes        = Container(total: uint8, enrs: List[ByteList, max_length=32])
 * `enrs`: List of byte strings, each of which is an RLP encoded ENR record.
     * Individual ENR records **MUST** correspond to one of the requested distances.
     * It is invalid to return multiple ENR records for the same `node_id`.
+    * The ENR record of the requesting & responding node **SHOULD** be filtered out of the list.
```

Comment:
```suggestion
    * The ENR record of the requesting node **SHOULD** be filtered out of the list.
```
I think only the requesting node should be mentioned here, as else it contradicts the above functionality of "The distance of `0` indicates a request for the recipient's own ENR record."

And for any other distance than `0`, it would be an invalid response anyhow if the responding node's ENR gets returned.